src/features/language/LanguageProvider.tsx

```diff
@@ -34,7 +34,7 @@ export const LanguageProvider = ({ children }: PropsWithChildren) => {
     const localStorageValue = localStorage.getItem(localStorageKey);
     const urlValue = getLanguageQueryParam();
 
-    const newLanguage = urlValue ?? localStorageValue ?? profile.profileSettingPreference.language;
+    const newLanguage = urlValue ?? localStorageValue ?? profile.profileSettingPreference.language ?? 'nb';
```

Comment:
Would you mind also updating the `IProfileSettingPreference.language` type to be optional so it reflects reality?

Reply:
Ah of course :smile: